app/upgrades/v13/upgrade_test.go

```diff
@@ -55,8 +54,6 @@ func (suite *UpgradeTestSuite) TestUpgrade() {
 		{
 			"Test that the upgrade succeeds",
 			func() {
-				acc := suite.App.AccountKeeper.GetAccount(suite.Ctx, ibc_hooks.WasmHookModuleAccountAddr)
-				suite.App.AccountKeeper.RemoveAccount(suite.Ctx, acc)
```

Comment:
Can we comment these lines, as indication to future us that they existed?

Reply:
Added with comments